RandLAPACK/comps/rl_orth.hh

```diff
@@ -72,7 +72,7 @@ int CholQRQ<T>::call(
     T* A
 ){
 
-    T* A_gram  = ( T * ) calloc( k * k, sizeof( T ) );
+    T* A_gram  = new T[k * k]();
```

Comment:
I think the standard syntax here is ``new T[k*k]{}`` (with braces, rather than parenthesis).

Reply:
Looks like you used parens in many places. Let's not bother changing for now. It could be that it makes no difference.

Reply:
Pretty much no diff for our purposes.